### ISSUE TITLE
Not taking commands which has argument in it

### ISSUE THREAD

**Issue by Bhupesh92** [4iar/lldb-write#1]
HI,

I am trying to execute following command to save info inside a file. But it is giving error.

command: 
 write $FILE_NAME image lookup --address $SYMBOL_ADDRRESS

Error:
usage: write [-h] filename command [command ...]
write: error: unrecognized arguments: --address 0x100004AAC

**Comment by kadirkelesoglu** [comment]
Hi,

I had the similar error and using double quotes for arguments worked for me.

Command getting error: write 770-stat.txt sos DumpHeap -stat
Error: write: error: unrecognized arguments: -stat

Command without error: write 770-stat.txt sos DumpHeap "-stat"

**Comment by dmetzgar** [comment]
It seems very particular. Here's how I have to do more than one argument:
```
write blah.txt sos ClrStack "-i" "-a"
```

That lets the command run but unfortunately the output from the command is not captured in the text file.

**Comment by 4iar** [comment]
Thanks for the info @dmetzgar! I have added support for lldb commands that have arguments. Will merge it later but feel free to give it a try in the meantime (in 
364950f or the simplify-argument-parsing branch)

Example (I couldn't run `sos` so used `image` instead):

```
(lldb) write imglookup2.txt image lookup --address 0x0000000100010000
      Address: dyld[0x0000000000000000] (dyld.__TEXT + 0)
      Summary: 
(lldb) platform shell cat imglookup2.txt
(lldb) image lookup --address 0x0000000100010000

      Address: dyld[0x0000000000000000] (dyld.__TEXT + 0)
      Summary: 
```

**Comment by 4iar** [comment]
Should be fixed by #7 (just merged to master)
